push node to testnet doc (#467)
ChainSafe/ethermint: docs/quickstart/deploy_node_on_existing_testnet.md

<!--
order: 9
-->

> # Deploy node to existing testnet

Reviewer: I'd move this to document under `/guides`

> Learn how to deploy a node to Digital Ocean and connect to public testnet

Reviewer: ```suggestion
Learn how to deploy and connect a node in the cloud to public testnet. {synopsis}
```

## Pre-requisite Readings

- [Install Ethermint](./installation.md) {prereq}
- [Start Testnet](./testnet.md) {prereq}

> - [Deploy Testnet to DigitalOcean](./testnet_on_digitalocean.md) {prereq}

Reviewer: update link with suggestion

## Deploy node to Digital Ocean

### Create a Droplet

> Create a new droplet using the same steps in [Deploy Testnet to DigitalOcean](./testnet_on_digitalocean.md). 

Reviewer: ditto link update

Once this new droplet is created make sure to get its IP address to be used in the next steps.

### Connect to Droplet

Click on the started Droplet, and you'll see details about it. At the moment, we're interested in the IP address - this is the address that the Droplet is at on the internet.

To access it, we'll need to connect to it using our previously created private key. From the same folder as that private key, run:

```bash
$ ssh -i digital-ocean-key root@<IP_ADDRESS>
```

Now you are connected to the droplet. 

### Install Ethermint

Clone and build Ethermint in the droplet using `git`:

```bash

> git clone https://github.com/ChainSafe/ethermint.git

Reviewer: you can use `go get` or `go install` instead of cloning the repo

cd ethermint
make install
```

Check that the binaries have been successfuly installed:

```bash
ethermintd -h
ethermintcli -h
```

#### Copy the Genesis File

To connect the node to the existing testnet, fetch the testnet's `genesis.json` file and copy it into the new droplets config directory (i.e `$HOME/.ethermintd/config/genesis.json`).

To do this ssh into both the testnet droplet and the new node droplet. 

On your local machine copy the genesis.json file from the testnet droplet to the new droplet using 

> ```bash

Reviewer: ```suggestion

```bash
```
